tests/libvmi/network.go

```diff
@@ -32,19 +32,30 @@ func WithInterface(iface kvirtv1.Interface) Option {
 	}
 }
 
+// WithPorts set the openned ports for all the network interfaces in the VMI
```

Comment:
It looks weird to open the same port for all the networks. Can we stick to `InterfaceDeviceWithMasqueradeBinding` and define the ports only there?

Reply:
Done